lib/ansible/modules/remote_management/cpm/cpm_status.py

```diff
@@ -0,0 +1,171 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+#
+# (C) 2018 Red Hat Inc.
+# Copyright (C) 2018 Western Telematic Inc.
+#
+# GNU General Public License v3.0+
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+# Module to execute CPM Status Commands on WTI OOB and PDU devices.
+# CPM remote_management
+#
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = """
+---
+module: cpm_status
+version_added: "2.7"
+author: "Western Telematic Inc. (@wtinetworkgear)"
+short_description: Get status and parameters from WTI OOB and PDU devices
```

Comment:
This should be a [lookup plugin](https://docs.ansible.com/ansible/latest/dev_guide/developing_plugins.html#lookup-plugins), not a module. Lookup plugins fetch data from external resources but do not make changes. Modules are only for performing create, update, or delete operations.